src/containers/SnippetArea.jsx

```diff
@@ -114,7 +121,13 @@ export class SnippetArea extends React.Component {
   }
 
   handleSaveAs() {
-    const { appState } = this.props;
+    const { snippetTitle, appState } = this.props;
+    if (!snippetTitle) {
+      this.setState({ titleErrorText: 'This field is required' });
+      this.showSnackbar('Please populate all required fields before saving.');
```

Comment:
Since we know which field it is, why not specifically reference the title field in the snackbar error?